library/src/test/java/com/datatorrent/lib/io/fs/HDFSOutputOperatorTest.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.datatorrent.lib.io.fs;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+public class HDFSOutputOperatorTest extends AbstractFileOutputOperatorTest
+{
+
+  /**
+   * Test file rollover in case of idle windows
+   * 
+   * @throws IOException
+   */
+  @Test
+  public void testIdleWindowsFinalize() throws IOException
+  {
+    HDFSOutputOperator writer = new HDFSOutputOperator();
+    writer.setFileName("output.txt");
+    writer.setFilePath(testMeta.getDir());
+    writer.setAlwaysWriteToTmp(false);
+    writer.setMaxIdleWindows(5);
+    writer.setup(testMeta.testOperatorContext);
+
+    String[][] tuples = {{"0a", "0b" }, {"1a", "1b" }, {}, {}, {}, {}, {"6a", "6b" }, {"7a", "7b" }, {}, {}, {},
```

Comment:
I tried to cover different scenarios with long input. Hence, would like to keep it unchanged.